src/main/api/ledger.ts

```diff
@@ -110,14 +111,55 @@ const signBtcTxInLedger = async (
   }
 }
 
+const signBNBTxInLedger = async (ledgerTxInfo: LedgerBNBTxInfo): Promise<E.Either<LedgerErrorId, string>> => {
+  const {
+    client,
+    sender,
+    asset,
+    amount,
+    recipient,
+    memo,
+    handleLedgerPresign,
+    handleLedgerVerifySuccess,
+    handleLedgerVerifyFailed
+  } = ledgerTxInfo
+  // await client.initChain()
+  const transport = await TransportNodeHid.open('')
+  const ledgerApp = new LedgerAppBNB(transport)
+  const derive_path = getBNBDerivePath(0)
+  client.useLedgerSigningDelegate(
+    ledgerApp,
+    () => {
+      if (handleLedgerPresign) handleLedgerPresign()
+    },
+    () => {
+      if (handleLedgerVerifySuccess) handleLedgerVerifySuccess()
+    },
+    (error) => {
+      if (handleLedgerVerifyFailed) handleLedgerVerifyFailed(getErrorId(error.message, error.statusText))
+    },
+    derive_path
+  )
+  const transferResult = await client.transfer(
```

Comment:
We have to update `xchain-binance` to provide a custom `transfer` function (_as you already mentioned it in Slack_). Currently it still [uses a private key of keystore](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-binance/src/client.ts#L329). We will try it to do it similar as we have already done it with xchain-btc: Providing custom functions for using Ledger only.